config/routes.rb

```diff
@@ -103,6 +103,8 @@
     put 'users/invite_users' => 'users#invite_users', as: :invite_users
 
     post 'search/users' => 'posts#search', as: :user_search
+    get  'search/new_user' => 'posts#new_user', as: :new_user
+    post  'search/new_user' => 'posts#change_owner_new_user'
```

Comment:
This was a little confusing to me, i expected the url to be `/posts/new_user` since thats where the method is.